packages/grid/stories/grid.doc.stories.mdx

```diff
@@ -66,43 +85,47 @@ You can easily create any project-specific column types using the same approach.
 
 Columns can be grouped using the `ColumnGroup` component.
 
+Note: whenever a `ColumnGroup` component is present, all other grid's children will also need to be wrapped within a `ColumnGroup` component.
+
 <Canvas>
   <Story id="grid-new-grid--column-groups" decorators={[withFlexGap]} />
 </Canvas>
 
 ### Rows
 
-Grid rows are created by providing an array of objects in the `rowData` property of the grid.
+The grid rows are created by providing an array of objects in the `rowData` property of the grid.
 `rowData` also accepts sparse arrays.
```

Comment:
Do you know what "sparse array" supposed to mean in this context? We might want to reword this, I have no idea what it meant.

Reply:
Maybe rephrase it so it just says that the shape of the objects in the array can be anything?